src/FileLocator/DefaultFileLocator.php

```diff
@@ -48,15 +48,14 @@ public function locate(string $name, string $currentPath = null): string
 
     private function isAbsolutePath(string $file): bool
     {
-        return ($file[0] === '/'
+        return $file[0] === '/'
```

Comment:
I would keep those parenthesis, they are here for readability

Reply:
When I read some code in (...) I suppose it's required and has special meaning (otherwise every code or condition would have it). This is not the case.

Reply:
I genuinely find it easier to read something like

```php
return (
    'foo' === $foo
    && 'bar === $bar
);
```

than:

```php
return 'foo' === $foo
    && 'bar === $bar;
```

(not the best example as in that case it could be easily inlined though).

That said we might just agree to disagree, I think it's very subjective for that one

Reply:
As you wish :bowing_man: